provisioning/image_cache.go

```diff
@@ -138,11 +167,17 @@ func newImageCacheContainers(images *Images, proxy *osconfigv1.Proxy) []corev1.C
 }
 
 func newImageCachePodTemplateSpec(info *ProvisioningInfo) (*corev1.PodTemplateSpec, error) {
-	cacheConfig, err := imageCacheConfig(info.Namespace, info.ProvConfig.Spec)
+	imageURL, err := imageCacheConfig(info.Namespace, info.ProvConfig.Spec)
 	if err != nil {
 		return nil, err
 	}
-	info.ProvConfig.Spec = *cacheConfig
```

Comment:
Glad we got rid of this: https://github.com/openshift/cluster-baremetal-operator/pull/148#discussion_r674993687

Reply:
Thanks for the context. This line was a head scratcher to me too. I was worried about imagecache changing the spec also.